src/__tests__/lib/exampleData.js

```diff
@@ -88,12 +94,12 @@ const userOrBotProperties = ({ name: _name }) => {
     full_name: `${capsName} User`,
     is_admin: false,
     timezone: 'UTC',
-    user_id: randUserId(),
+    user_id: user_id ?? randUserId(),
   });
 };
 
 /** Beware! These values may not be representative. */
-export const makeUser = (args: { name?: string } = {}): User =>
+export const makeUser = (args: UserOrBotPropertiesArgs = NULL_OBJECT): User =>
```

Comment:
**@&zwj;gnprice** wrote:
> What's reasoning for `NULL_OBJECT` here? `{}` is shorter and is one fewer indirection to understand.

`{}` is not exact, and so doesn't typecheck (Flow issue 2386 and/or 2977); and `Object.freeze({})`, which is the usual way to get an explicitly-exact empty object, is longer.

... I thought we had some other examples of using `NULL_OBJECT` as a default parameter in this context, but apparently not. What would you prefer?

Reply:
Hmm, I see. Seems an OK workaround for that, then.